.github/workflows/package_for_test.yml

```diff
@@ -110,6 +110,15 @@ jobs:
         env:
           CI: false
 
+      - name: Write electron-build Test yml
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const fs = require('node:fs')
+            const prodYmlPath = 'packages/neuron-wallet/electron-builder.yml'
+            const testYmlPath = 'packages/neuron-wallet/electron-builder.test.yml'
+            fs.writeFileSync(testYmlPath, fs.readFileSync(prodYmlPath).toString().replace('asar: true', 'asar: false'))
```

Comment:
It looks good, but I wonder why not override the original `yaml` file. Only one file need to be updated by doing so.